packages/core/src/App/Containers/Layout/header/default-header.jsx

```diff
@@ -71,6 +73,13 @@ const DefaultHeader = ({
     }, [removeUpdateNotification]);
 
     const onClickDeposit = () => history.push(routes.cashier_deposit);
+
+    const verification_status = populateVerificationStatus(account_status);
+    const { has_poa, has_poi } = verification_status;
+    const onClickUpgrade = () => {
+        // TODO: here we should open POA and POI modal
```

Comment:
TODO declaration is needed for future functionality or we can remove it?

Reply:
It is related to modal that Amina has implemented, After we merged that PR I should change this TODO